plugins/spamassassin.js

```diff
@@ -309,8 +309,8 @@ exports.get_spamd_socket = function (next, conn, headers) {
         else {
             conn.logerror(plugin, 'timeout waiting for results');
         }
-        socket.end();
-        return next();
+        socket.destroy()
+        return next(CONT, 'timeout error');
```

Comment:
```suggestion
       next(CONT, 'timeout');
```

Reply:
• A timeout isn't necessarily an error. 
• the return at EOF is superfluous, right?